index.html

```diff
@@ -14,17 +14,21 @@
     <main>
         <div class="intro">
             <header>
-              <div class="nav_conatiner">
+              <div class="nav_container">
                 <nav class="nav-bar">
-                  <a href="#"><h1 class="logo">My Logo</h1></a>
+                  <a href="#"><h3 class="logo">My Logo</h3></a>
+                  <div class="humburger">
+                    <button id="open-menu-btn"><i class="uil uil-bars"></i></button>
+                    <button id="close-menu-btn"><i class="uil uil-multiply"></i></button>
+                  </div>
                   <ul class="nav-links">
-                    <li><a class="item" href="#">Portfolio</a></li>
-                    <li><a class="item" href="#">About</a></li>
+                    <li><a class="item" href="#profile">Portfolio</a></li>
+                    <li><a class="item" href="#about">About</a></li>
                     <li class="contact">
-                      <a class="item" href="#">Contact</a>
+                      <a class="item" href="#contact">Contact</a>
```

Comment:
- [ ] Kindly there's a horizontal scrolling on your page, the close button is out of the page, kindly fix it to match the figma design.
![image](https://user-images.githubusercontent.com/35535130/173465663-c3d9a925-656d-4d0e-8bb4-231fff415c07.png)
![image](https://user-images.githubusercontent.com/35535130/173465687-3d3427cc-6f1a-4d4c-93a1-8e6983d0b66d.png)